lib/rules/no-descending-specificity/__tests__/index.js

```diff
@@ -147,3 +147,15 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  syntax: "scss",
+  config: [true],
+
+  accept: [
+    {
+      code: ".foo { &--a, &--a#{&}--b {  div { padding: 10px; } } }"
```

Comment:
As [per the conventions](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#example-patterns):

> Use the minimum amount of code possible to communicate the pattern e.g. if the rule targets selectors then use an empty rule e.g. {}.

Can we use the following please?:

```js
.foo { &--a, &--a#{&}--b { div {} } }